### ISSUE TITLE
GIST test uses a lot of memory

### ISSUE THREAD

**Issue by drroe** [Amber-MD/cpptraj#209]
This is what I think is causing the GIST test being run in #208 to fail. A typical test uses on the order of MB. The GIST test is calling for almost 6 GB (output from valgrind below):
```
==28652==   total heap usage: 91,788 allocs, 91,788 frees, 5,827,290,613 bytes allocated
```
The GIST test current uses a 10 frame trajectory of a small peptide (5293 atoms, 1704 res, 1692 mol, 1691 solvent) with the following script:
```
parm ../tz2.ortho.parm7
trajin ../tz2.ortho.nc 1 10
autoimage origin
gist doorder doeij refdens 0.033422885325 gridcntr 1.5 1.0 0.0 \
  griddim 34 44 36 gridspacn 0.50 out gist.out
go
```
The culprit seems to be the `doeij` keyword; without it memory usage is greatly reduced:
```
==31073==   total heap usage: 26,386 allocs, 26,386 frees, 24,929,574 bytes allocated
```
@cnnguyen This seems to be because the `ww_Eij_` array needs to be an upper triangle matrix with NX * NY * NZ elements (i.e. the total number of voxels) - is that correct? If so then I think we should disable `doeij` for the existing test, and test the option separately on a much smaller system. Do either you or @gosldorf have a good and small test case for `doeij`? Otherwise I will try to devise a reasonable one. Thanks!

**Comment by cnnguyen** [comment]
Hi Dan,

It is correct that the energy matrix takes up a lot of disk space. That option should have not been made available at first place. It's even less useful for non-experience Gist users. But well, if because it is already there, we can create a smaller test case. Why don't we reduce the gist box to something like 5x5x5? 

Thanks
Crystal
________________________________________

**Comment by drroe** [comment]
@cnnguyen I've actually done this as part of #208. I'm just trying to iron out some of the windows-introduced complications of the appveyor build (it likes to print 3-digit exponents but they should only be 2, causing spurious test failure).

**Comment by drroe** [comment]
Addressed by #208
